spec/requests/v1/report_templates/delete_logo_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'DELETE v1/sessions/:session_id/report_templates/:id/remove_logo', type: :request do
+  let(:intervention) { create(:intervention) }
+  let(:session) { create(:session, intervention: intervention) }
+  let(:user) { create(:user, :researcher, :confirmed) }
+
+  let(:headers) { user.create_new_auth_token }
+
+  let(:request) do
+    delete v1_session_report_template_remove_logo_path(
+      session_id: session.id, report_template_id: report_template.id
+    ), headers: headers
+  end
+
+  let!(:report_template) { create(:report_template, :with_logo, session: session) }
+
+  shared_examples 'can delete the report logo' do
+    it 'returns the response code for no content' do
+      request
+      expect(response).to have_http_status(:no_content)
+    end
+
+    it 'removes the attachment' do
+      expect { request }.to change(ActiveStorage::Attachment, :count).by(-1)
+    end
+  end
```

Comment:
let's extract this as a global shared example. Define in one file in `spec/support/shared_examples` 
https://rspec.info/features/3-12/rspec-core/example-groups/shared-examples/